frontends/search/src/components/ChunkMetadataDisplay.tsx

```diff
@@ -314,10 +314,6 @@ const ChunkMetadataDisplay = (props: ChunkMetadataDisplayProps) => {
             innerHTML={sanitizeHtml(
               props.chunk.chunk_html !== undefined
                 ? props.chunk.chunk_html
```

Comment:
I believe this should get the same replace as in the `ScoreChunk` file. Will ship it real quick.